packages/itwinui-css/src/input/input.scss

```diff
@@ -81,3 +81,12 @@
     resize: none;
   }
 }
+
+@mixin iui-search-box {
+  &::-webkit-search-decoration,
+  &::-webkit-search-cancel-button,
+  &::-webkit-search-results-button,
+  &::-webkit-search-results-decoration {
+    appearance: none;
```

Comment:
if we are removing the default browser functionality of clearing the input, then we should show an example of replacing it with a X button that also clears the input

Reply:
Removed this functionality and using type='text' instead.